assets/src/scripts/_utils.js

```diff
@@ -120,3 +122,22 @@ export function formatDate(date) {
 export function hasComment(outputName) {
   return outputComments.value[outputName].trim() !== "";
 }
+
+/**
+ * @param {string} ext
+ */
+export function highlightJsName(ext) {
```

Comment:
Is there anyway for highlight to guess the language based on file extension?

Reply:
I believe it will try to guess based on the extension matching an item from this list:
https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md

Reply:
Testing with an example `.scss` file, and it highlights

![CleanShot 2023-10-24 at 15 28 17@2x](https://github.com/opensafely-core/sacro/assets/24863179/f674fd5d-96e6-44c8-abba-22a11bc17e8b)